README.md

```diff
@@ -15,7 +15,7 @@ It helps you sync 2 directories at a time. There are two types.
 
 Master
 ------
-Contents of this directory are moved to Slave for writing and updating. If this folder has a path that Slave doesn't then the path on Slave is deleted.
+Contents of this directory are moved to Slave for writing and updating. If Slave has a folder has a path that this doesn't then the path on Slave is deleted.
```

Comment:
That logic is correct Phil. And Dan that wording is a little off. I'm going to leave that wording on the deleting alone. Maybe something like "If Slave has a folder that this directory doesn't: then it is deleted." What do you guys think?